README.md

```diff
@@ -1070,6 +1071,26 @@ params do
 end
 ```
 
+### Structures Implicit Default Value
```

Comment:
Reading this it feels like it's wrong. Yes, it works this way. But if I send a `nil`, why am I getting a non-nil?